front/app/components/admin/GraphCards/CommentsByTimeCard/useCommentsByTime/translations.ts

```diff
@@ -0,0 +1,26 @@
+// i18n
+import messages from './messages';
+import moduleMessages from 'components/admin/GraphCards/messages';
+import dashboardMessages from 'containers/Admin/dashboard/messages';
+import { getTimePeriodTranslations } from 'components/admin/GraphCards/_utils/resolution';
+
+// typings
+import { FormatMessage } from 'typings';
+
+export interface Translations {
```

Comment:
Small improvement but you could extend these `Translations` from the `TimePeriodTranslations` interface in `components/admin/GraphCards/_utils/resolution`